rooms/views.py

```diff
@@ -1,3 +1,60 @@
-from django.shortcuts import render
+import json, datetime
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+from django.db.models import Avg, Count
+
+from .models import *
+
+class RoomDetailView(View):
+    def get(self, request, room_id):
+        try:
+            room = Room.objects.get(id = room_id)
+            average_rating = room.review_set.aggregate(average = Avg('rating'))['average']
```

Comment:
```suggestion
            room           = Room.objects.get(id = room_id)
            average_rating = room.review_set.aggregate(average = Avg('rating'))['average']
```
데이터베이스에 없는 room_id가 왔을 때의 예외처리 추가해 주세요.

Reply:
네엡! 😊